helm/fdb-operator/templates/manager/deployment.yaml

```diff
@@ -29,12 +29,68 @@ spec:
       - name: tmp
         emptyDir: {}
       - name: logs
-        emptyDir: {}        
+        emptyDir: {}
+      - name: fdb-binaries
+        emptyDir: {}
+      initContainers:
+        - name: foundationdb-kubernetes-init-6-1
+          image: foundationdb/foundationdb-kubernetes-sidecar:6.1.13-1
+          args:
+            - "--copy-library"
+            - "6.1"
+            - "--copy-binary"
+            - "fdbcli"
+            - "--copy-binary"
+            - "fdbbackup"
+            - "--copy-binary"
+            - "fdbrestore"
+            - "--output-dir"
+            - "/var/output-files/6.1.12"
```

Comment:
If we copied the files to `/var/output-files/6.1` we wouldn't ned to copy them around in the manager?

Reply:
We still would, because the directory format is different. I plan to file a follow-up issue on changing the behavior of the sidecar to make the copying in the manager unnecessary, but I don't want to block on that because changes to the sidecar have a longer release cycle.